templates/newmap.html

```diff
@@ -116,7 +116,10 @@
             var minutes = parseInt(diff / 60);
             var seconds = parseInt(diff - (minutes * 60));
             var expires_at = minutes + 'm ' + seconds + 's';
-            return '<b>#' + item.pokemon_id + ' ' + item.name + '</b> ' + expires_at;
```

Comment:
as said in discord I think it would be fancier to have a
```
var label = '<b>#' + item.pokemon_id + ' ' + item.name + '</b> ' + expires_at;
if (item.atk != undefined) {
    label += '</br> IV:' + item.atk + '/' + item.def + '/' + item.sta + '</br>' + item.move1 + '</br>' + item.move2;
}
return label;
```